SPECS/python-botocore/python-botocore.spec

```diff
@@ -0,0 +1,1860 @@
+Summary:        Low-level, data-driven core of boto 3
+Name:           python-%{pypi_name}
+# NOTICE - Updating this package requires updating python-boto3
+Version:        1.24.33
+Release:        2%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/boto/botocore
+Source0:        https://github.com/boto/botocore/archive/refs/tags/%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
+%global pypi_name botocore
+# Replace removed inspect.formatargspec with inspect.signature
+# This is needed for compatibility with Python 3.11 and
+# can be removed once upstream PR is merged.
+Patch2507:      https://github.com/boto/botocore/pull/2507.patch
+BuildArch:      noarch
+
+BuildRequires:  pyproject-rpm-macros
+BuildRequires:  python3
+BuildRequires:  python3-pip
```

Comment:
Pip shouldn't be necessary for non-check builds, please gate appropriately

Reply:
Done